x-pack/plugins/transform/server/routes/api/privileges.ts

```diff
@@ -38,40 +39,26 @@ export function registerPrivilegesRoute({ router, license }: RouteDependencies)
           },
         });
 
-      // Find missing cluster privileges and set overall app privileges
-      privilegesResult.missingPrivileges.cluster = extractMissingPrivileges(cluster);
-      privilegesResult.hasAllPrivileges = hasAllPrivileges;
-
       // Get all index privileges the user has
       const { indices } = await esClient.asCurrentUser.security.getUserPrivileges();
```

Comment:
nit pick, the calls to `security.hasPrivileges` and `security.getUserPrivileges` could be put in a `Promise.all`